includes/orders/classes/class-order-item.php

```diff
@@ -232,7 +232,15 @@ public function get_fees() {
 	 * @return string The product name including any price ID name.
 	 */
 	public function get_order_item_name() {
-		return $this->product_name;
+		return is_admin() ?
```

Comment:
Note: `edd_is_admin_page()` returns `false` for the refunds table so I stuck with `is_admin()` instead.